README.md

```diff
@@ -201,17 +196,11 @@ Example:
 
 <!-- AUTHORS -->
 
-## 👥 Authors <a name="authors"></a>
+## 👥 Authors <a name="authors">Edgard Soza</a>
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
```

Comment:
I highlighted this piece of code because I couldn't highlight the relevant piece of code in this file. Kindly make the following changes:

- [ ] For the **_Key Features_** section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍 
- [ ] Kindly add a **_Future Feature_** Section that lists 1-3 features that can be implemented in the future. Adding this information makes it easy for prospective collaborators to know how to contribute to this project. If you also want to work further on this project in the future, these features will give you a headstart. 👍 
- [ ] For the **_Prerequisite_** Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
- Installation of `Git`
- Installation of `NPM`
- A working browser
- Internet connection
- A code editor like `Visual Studio Code`
- Installation of `Live Server` plugging in Visual Studio Code
- [ ] For the Setup section, kindly add instructions for cloning your repository on a local machine 👍 
- [ ] For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code.
- [ ] For the MIT section, kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE

https://choosealicense.com/licenses/mit/

#### Kindly implement all the changes listed above. It will make your _**README.md**_ file look professional and it will give you an idea of how to set _**README.md**_ files for most of the projects you will execute as a Microverse student. 👍 💻

Reply:
@Gambit142 all done.

Reply:
@Gambit142 should I request other review in dashboard?